com.unity.render-pipelines.universal/Shaders/Terrain/WavingGrassBillboard.shader

```diff
@@ -32,6 +32,7 @@ Shader "Hidden/TerrainEngine/Details/UniversalPipeline/BillboardWavingDoublePass
             #pragma multi_compile _ _ADDITIONAL_LIGHT_SHADOWS
             #pragma multi_compile _ _SHADOWS_SOFT
             #pragma multi_compile _ _MIXED_LIGHTING_SUBTRACTIVE
+            #pragma multi_compile _ _SCREEN_SPACE_OCCLUSION
```

Comment:
Does it even work with billboards since we reconstruct normal from depth?

Reply:
Nope! Didn't check that well enough. Removed!